sdk/storage/azure-storage-files-shares/CHANGELOG.md

```diff
@@ -12,6 +12,8 @@
   - ShareClient::BreakLease
   - ShareClient::RenewLease
 - Move File SAS into `Azure::Storage::Sas` namespace.
+- Replaced all transactional content MD5/CRC64 with `ContentHash` struct.
+- `FileShareHttpHeaders` is renamed to `ShareFileHttpHeaders`, and member `std::string ContentMd5` is changed to `Storage::ContentHash ContentHash`.
```

Comment:
Is this change relevant to this PR?

Reply:
This is relevant for this RP due to alignment to .NET SDK for HttpHeaders, including the name of the HttpHeaders and it's content.

Reply:
ok